src/components/home/elements/TransactionRecords.vue

```diff
@@ -227,15 +226,41 @@ export default {
             const d = new Date(date / 1e6)
             return monthNames[d.getMonth()] + ', ' + d.getFullYear()
         },
-        getSendExecuteContractTxType(functionIndex, Data) {
+        parseExecution(functionIndex, Data) {
```

Comment:
`functionIndex` is not used in `parseExecution(...)`. As Token added in watching list, we can known "Contract Type" from local storage by Token ID (or Contract ID). To make this function easy to extend and maintain, suggest using "Contract Type" + "Function Index" to find out "Function Type" instead of 
simply using `functionData.length`.